documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/template-registration.rst

```diff
@@ -0,0 +1,36 @@
+.. _applying-lumbar-registration-algorithm:
+
+Applying the registration algorithm
+###################################
+
+To apply the registration algorithm, the following command is used:
+
+.. code:: sh
+
+   sct_register_to_template -i t2_crop.nii.gz \
+                            -s t2_crop_seg.nii.gz \
+                            -ldisc t2_crop_labels.nii.gz \
+                            -c t2 -qc ~/qc_singleSubj \
+                            -param step=1,type=seg,algo=centermassrot:step=2,type=seg,algo=bsplinesyn,metric=MeanSquares,iter=3,slicewise=0:step=3,type=im,algo=syn,metric=CC,iter=3,slicewise=0
```

Comment:
@jcohenadad What methodology did you use to select these parameters when [responding on the forum](https://forum.spinalcordmri.org/t/labels-for-lumbar-spinal-level/737/8?u=joshuacwnewton)? 

Should we repeat these parameters as-is, or should we start out with the default parameters (and then instruct the user to tweak parameters for their data and/or post on the forum)?

Reply:
Feedback given in SCT meeting -- will repeat here later.

Reply:
the syn step is WAAAY better! thank you for tweaking this!